src/internal_macros.rs

```diff
@@ -169,7 +169,7 @@ macro_rules! display_from_debug {
 }
 
 #[cfg(test)]
-macro_rules! hex_script (($s:expr) => ($crate::blockdata::script::Script::from(<Vec<u8> as $crate::hashes::hex::FromHex>::from_hex($s).unwrap())));
+macro_rules! hex_script (($s:expr) => (<$crate::Script as ::std::str::FromStr>::from_str($s).unwrap()));
```

Comment:
I was wondering about the reason to have this macro, is it to hide the unwrap?

isn't slightly better to use in test `Script::from_hex("0100").unwrap()` instead of a macro call that hides the method called and it is usable only internally?

Reply:
it doesn't matter because this is for test code only

Reply:
The macro used to hide a pile of complicated stuff. Now it only hides the unwrap. At this point you're right that the macro serves little purpose and basically just obfuscates ... but removing it would've made the diff much bigger.